yarn.lock

```diff
@@ -456,6 +456,13 @@
   resolved "https://registry.yarnpkg.com/@types/markdown-it/-/markdown-it-0.0.4.tgz#c5f67365916044b342dae8d702724788ba0b5b74"
   integrity sha512-FWR7QB7EqBRq1s9BMk0ccOSOuRLfVEWYpHQYpFPaXtCoqN6dJx2ttdsdQbUxLLnAlKpYeVjveGGhQ3583TTa7g==
 
+"@types/moment-duration-format@^2.2.2":
```

Comment:
double-check with @anoek  but I don't think we're supposed to submit edits to yarn.lock and rather it should auto-build on compile? sorry if that's mistaken

Reply:
Nope commits to `yarn.lock` are expected

Reply:
Thanks